topics/single-cell/tutorials/scrna-case_JUPYTER-trajectories/tutorial.md

```diff
@@ -72,23 +85,39 @@ From now on, you can view this tutorial in the Jupyter notebook, which will allo
 pip install scanpy
 ```
 ```python
-pip install fa2
-```
-```python
 pip install igraph
 ```
 ```python
 pip install louvain
 ```
 ```python
-
 import numpy as np
 import pandas as pd
 import matplotlib.pyplot as pl
 from matplotlib import rcParams
 import scanpy as sc
 ```
 
+And lastly, the trickiest one: ForceAtlas2. The original `fa2` only runs on Python <3.8, but there is another, maintained version - `fa2_modified` which can be installed on Python 3.9+. 
```

Comment:
Is it an option to install scanpy with python37 into a separate environment?

`conda create --name scanpy python=3.7 scanpy`

Reply:
Thanks, but it works well with the current method and is a more general solution which I hope will prove useful to a broader range of people